src/main/controllers/SearchOptionController.ts

```diff
@@ -9,5 +9,8 @@ export default class SearchOptionController {
     if (req.body['find-choice'] === 'search') {
       res.redirect('search');
     }
+    if (req.body['find-choice'] === 'find') {
```

Comment:
theres also the unit test for this path too in SearchOptionController.test.ts

Reply:
Added unit test in for this